timely-beliefs-master/timely_beliefs/visualization/ridgeline.py

```diff
@@ -0,0 +1,130 @@
+from bokeh.io import output_file, show
+from bokeh.models import ColumnDataSource, FixedTicker, PrintfTickFormatter
+from bokeh.plotting import figure
+from bokeh.sampledata.perceptions import probly
+from bokeh.palettes import cividis, viridis
+import colorcet as cc
+import datetime
+import csv
+import math
+import numpy as np
+import scipy.stats as stats
+import pandas as pd
+
+
+def ridgeline_plot(date, csv_05, csv_095, output=False, start=0, end=168):

```

Comment:
Can you separate the i/o functionality from the plotting functionality? Right now the plotting function depends on a file system where csv files are stored. I'd recommend separately loading the two csv files into DataFrame or BeliefsDataFrame objects, and then passing those objects to the plotting function.